index.js

```diff
@@ -1,17 +1,235 @@
+const { google } = require('googleapis');
 const fs = require('fs');
-const path = require('path');
+const has = require('lodash.has');
+const { klona } = require('klona');
 
 module.exports = {
-  bundle: {
-    directory: 'modules',
-    modules: getBundleModuleNames()
+  improve: '@apostrophecms/form',
+  fields: {
+    add: {
+      googleSheetSubmissions: {
+        label: 'aposForm:activateGoogleSubmit',
+        type: 'boolean'
+      },
+      googleSpreadsheetId: {
+        label: 'aposForm:googleSheetId',
+        type: 'string',
+        required: true,
+        htmlHelp: 'aposForm:googleSheetIdHtmlHelp',
+        if: {
+          googleSheetSubmissions: true
+        }
+      },
+      googleSheetName: {
+        label: 'aposForm:googleSheetName',
+        type: 'string',
+        help: 'aposForm:googleSheetNameHelp',
+        if: {
+          googleSheetSubmissions: true
+        }
+      }
+    },
+    group: {
+      afterSubmit: {
+        fields: [
+          'googleSheetSubmissions',
+          'googleSpreadsheetId',
+          'googleSheetName'
+        ]
+      }
+    }
+  },
+  methods (self) {
+    return {
+      async sendToGoogle(req, form, formData) {
+        if (form.googleSheetSubmissions === true) {
+          // Don't modify the original, this would frustrate other custom
+          // submission handlers
+          const data = klona(formData);
+          const timeRegex = /^(.*?)T(.*?)(\..*)$/;
+          const timeFields = (new Date()).toISOString().match(timeRegex);
+
+          const dateColumn = self.options.dateColumnLabel || 'Date Submitted';
```

Comment:
This is new. l10n doesn't work well for this since it needs to be consistent, so it's a configurable option. It's possible this could also be a user facing field set

Reply:
Interesting, yeah I don't see a great way around it.

Reply:
Nor do we really need another approach since as you say it's for admins to read and they can put it in their preferred admin language with this option.